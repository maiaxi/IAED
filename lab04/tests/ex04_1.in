8 -1 3 6 -12 55 -9 7 -21
