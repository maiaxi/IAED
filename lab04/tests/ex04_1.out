-1 2 8 -4 51 42 49 28
